src/Workspaces/Remote/ServiceHub/Services/RemoteSymbolSearchUpdateEngine.cs

```diff
@@ -67,23 +68,23 @@ public Task<IList<ReferenceAssemblyWithTypeResult>> FindReferenceAssembliesWithT
 
         #region Messages to forward from here to VS
 
-        public Task LogExceptionAsync(string exception, string text)
-            => this.InvokeAsync(nameof(LogExceptionAsync), new object[] { exception, text }, CancellationToken.None);
+        public Task LogExceptionAsync(string exception, string text, CancellationToken cancellationToken)
+            => this.InvokeAsync(nameof(LogExceptionAsync), new object[] { exception, text }, cancellationToken);
```

Comment:
@jasonmalinowski this is the crux of it.  Heejae made individual RPCs cancellable a while back, but at that time just updated all callers to pass CancellationToken.None.  I've been slowly going through updating these features to do things properly and pass along their cancellation tokens so that we can be better citizens.  It's not really complex :)